app-modules/campaign/src/Filament/Blocks/ServiceRequestBlock.php

```diff
@@ -84,17 +85,13 @@ public function generateFields(string $fieldPrefix = ''): array
                     titleAttribute: 'name',
                     modifyQueryUsing: fn (Builder $query) => $query->orderBy('order'),
                 )
+                ->model(ServiceRequest::class)
                 ->label('Priority')
                 ->required()
                 ->exists((new ServiceRequestPriority())->getTable(), 'id'),
-            Select::make($fieldPrefix . 'type_id')
-                ->relationship('type', 'name')
-                ->preload()
-                ->label('Type')
-                ->required()
-                ->exists((new ServiceRequestType())->getTable(), 'id'),
```

Comment:
Why was this removed? Is this the cause of Service Request Campaigns not working?

If so, we still need to be able to define the type and priority here. So we will need to revert this and create a separate ticket for fixing this.

Unless it prevents the fix for Campaign Engagement images

Reply:
As discussed, we will create a seperate ticket to address fixing this more in-depth in a future iteration.